content/chainguard/libraries/java/overview.md

```diff
@@ -141,3 +106,49 @@ is strongly recommended.
 Alternatively, you can use the token for direct access from a build tool as
 discussed in [Build
 configuration](/chainguard/libraries/java/build-configuration/).
+
+<a name="java-repo-test">
```

Comment:
Where are you referencing the `a name`s?

FWIW I believe they were deprecated by W3C

Reply:
I changed to use `id` .. and I am linking to these two sections to each other in the context.